Modules/_zstd/decompressor.c

```diff
@@ -62,7 +62,7 @@ _get_DDict(ZstdDict *self)
 
 /* Set decompression parameters to decompression context */
 int
```

Comment:
```suggestion
static int
```

See above about static linkage.